storage/tianmu/common/common_definitions.cpp

```diff
@@ -32,6 +32,74 @@ void PushWarning(THD *thd, Sql_condition::enum_severity_level level, uint code,
   push_warning(thd, level, code, msg);
 }
 
+// Here for args int `type`, we do not use enum directly as it'll caused compiling failed for the dependent package.
```

Comment:
please add a TODO here, and change back to enum type later.

Reply:
OK, I had add it at the beginning of the comment.